src/workerd/io/actor-cache.c++

```diff
@@ -3362,4 +3362,8 @@ kj::Promise<void> ActorCacheInterface::waitForBookmark(kj::StringPtr bookmark) {
       Error, "This Durable Object's storage back-end does not implement point-in-time recovery.");
 }
 
+void ActorCacheInterface::ensureReplicas() {
+  JSG_FAIL_REQUIRE(Error, "This Durable Object's storage back-end does not support replication.");
```

Comment:
Nit: Can you add a comment along the lines of this being intended to be implemented internally

Reply:
https://github.com/cloudflare/workerd/compare/999831c699fd945aaddc50c446e07a6986d4a3b2..ced94358c1676c72530c24e3487170339ff98e8f